### DISCUSSION TITLE
mspider/intro/scenarios/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3366] [Announcements]
# mspider/intro/scenarios/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/intro/scenarios/

**Comment by kebe7jun** [comment]
可否增加传统微服务迁移的场景？

**Comment by kebe7jun** [comment]
还有多集群的场景。
